### ISSUE TITLE
armv7: Compilation error, missing RAM_SIZE

### ISSUE THREAD

**Issue by kemonats** [phoenix-rtos/phoenix-rtos-kernel#115]
### armv7m7-imx:
```
vm/page-nommu.c: In function '_page_init':
./hal/armv7/pmap.h:24:41: error: 'RAM_SIZE' undeclared (first use in this function)
   24 | #define VADDR_MAX       (VADDR_KERNEL + RAM_SIZE * 1024)
      |                                         ^~~~~~~~
```
### armv7m3-stm armv7m4-stm:
```
hal/armv7/_init-stm32.S: Assembler messages:
hal/armv7/_init-stm32.S:63: Error: invalid operands (*ABS* and *UND* sections) for `*'
```

**Comment by pawelpisarczyk** [comment]
I've asked Alek to correct this.

**Comment by agkaminski** [comment]
Fixed in a7c41d205ea53291eee9cecee24f6a07103018b2 and bd221df8c125f4d6206386035336ba0b679beb93 in phoenix-rtos-build
